components/Common/Banner/index.tsx

```diff
@@ -0,0 +1,21 @@
+import LocalizedLink from '@/components/LocalizedLink';
+import { ArrowUpRightIcon } from '@heroicons/react/24/outline';
+import styles from './index.module.scss';
+import type { FC } from 'react';
+
+type BannerProps = {
+  type: 'default' | 'error' | 'warning';
+  text: string;
+  url?: string;
+};
+
+const Banner: FC<BannerProps> = ({ type, text, url = '' }) => (
+  <div className={`${styles.banner} ${styles[type || 'default']}`}>
+    {(url.length > 0 && <LocalizedLink href={url}>{text}</LocalizedLink>) || (
+      <span>{text}</span>
```

Comment:
As I said there's no need for a `<span>` here imo

Reply:
for semantique it's better, I thinks

Reply:
I don't have strong opinion on this but...

I think `<span>` is used for grouping different parts of a text so it is redundant in this case.

Reply:
> for semantique it's better, I thinks

What do you mean by semantics? 😅

Reply:
> I think <span> is used for grouping different parts of a text so it is redundant in this case.

It's not even for that https://developer.mozilla.org/en-US/docs/Web/HTML/Element/span

It's often just to be able to separate or style different chunks of texts. Which we don't do here.

Reply:
`<p>` could be a better semantic tag

Reply:
No. P is for paragraphs, this is not a paragraph.

Please, read the docs and avoid random suggestions. There is no need for an extra HTML tag here.